gem/views.py

```diff
@@ -60,3 +62,53 @@ def form_valid(self, form):
         authed_user = authenticate(username=username, password=password)
         login(self.request, authed_user)
         return HttpResponseRedirect(form.cleaned_data.get('next', '/'))
+
+
+class GemRssFeed(Feed):
+    title = 'GEM Feed'
+    description = 'GEM Feed'
+    description_template = 'feed_description.html'
+
+    def __call__(self, request, *args, **kwargs):
+        self.base_url = '{0}://{1}'.format(request.scheme, request.get_host())
+        return super(GemRssFeed, self).__call__(request, *args, **kwargs)
+
+    def get_feed(self, obj, request):
+        feed = super(GemRssFeed, self).get_feed(obj, request)
+        # set language to Tagalog instead of settings.LANGUAGE_CODE
+        feed.feed['language'] = 'tl'
```

Comment:
:( sorry, this is only available in the new stuff that's unreleased yet.

For now you'll have to leave it as `settings.LANGUAGE_CODE`
We set this for all projects so it should work as expected.